.github/workflows/build.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
           directory: manifests
 
       - name: Build binary
-        run: CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' .
+        run: make binary-all
```

Comment:
Not sure how much parallelism the GHA workers would be able to handle, but how about adding at least `-j 2 ` here? 🤔

Reply:
sure can update with j2 and see if runs any better